firebase_admin/_messaging_encoder.py

```diff
@@ -324,7 +324,9 @@ def encode_android_notification(cls, notification):
 
         event_time = result.get('event_time')
         if event_time:
-            result['event_time'] = str(event_time.isoformat()) + 'Z'
+            if event_time.tzinfo is not None:
+                event_time = event_time.astimezone(datetime.timezone.utc)
+            result['event_time'] = event_time.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
```

Comment:
Please add a comment here explaining the logic and the rationale here.

Reply:
Updated! Thanks!